bestOfArtist.py

```diff
@@ -0,0 +1,108 @@
+import requests
+from bs4 import BeautifulSoup
+
+HEADERS_GET = {
+    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36',
+    'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8',
+    'Accept-Language': 'en-US,en;q=0.5',
+    'Accept-Encoding': 'gzip, deflate',
+    'DNT': '1',
+    'Connection': 'keep-alive',
+    'Upgrade-Insecure-Requests': '1'
+}
+METACRITIC_ALBUMS_TAG_ATTRS = {"class": "title brief_metascore"}
+RATE_YOUR_MUSIC_ALBUMS_TAG_ATTRS = {"class": "album"}
+RATE_YOUR_MUSIC_RATING_TAG_ATTRS = {"class": "disco_avg_rating"}
+METACRITIC_ACTIVE_TAG_ATTRS = {"class": "active"}
+
+
+def fromTopTen(artist, howMany):
+    toReturn = []
+    search = "https://www.google.com/search?q={artist}+albums+site:https://www.thetoptens.com/".format(
+        artist=artist)
+    res = requests.get(search, headers=HEADERS_GET).text
+    soup = BeautifulSoup(res, 'html.parser')
+
+    name = [tag.text for tag in soup.find_all(attrs={"class": "LC20lb DKV0Md"})]
+    if not name:
```

Comment:
change variable name tofirst_ result_name and add [0]

Reply:
if no result found u cant do [0], this is why I wrote it that way